include/rawcan_comm.hpp

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include <linux/can.h>
+
+namespace roboteq { 
+
+class rawcan_comm : public roboteq {
```

Comment:
i believe we renamed `roboteq` here to `i_comm`

Reply:
changed to i_comm